app/views/devise/passwords/edit.haml

```diff
@@ -38,7 +38,7 @@
                            aria:           {labelledby: "passwordConfirmationLabel"}
 
       = hidden_field(:user, :remember_me, value: 1)
-      = f.submit t('devise.passwords.edit.change_password'), class: "btn btn-block btn-default"
+      = f.submit t('devise.passwords.edit.change_password'), class: "btn btn-block btn-primary"
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.